build.gradle

```diff
@@ -34,29 +34,39 @@ buildScan {
 }
 
 String hailVersion = "devel"
-String[] deployedSparkVersions = ["2.0.2", "2.1.0"]
+String[] deployedSparkVersions = ["2.0.2"]
```

Comment:
should we add 2.2.0 to the deployed versions?

Reply:
I was going to do that after 2.2.0 was being tested so we're not deploying something (too) broken.

Reply:
ah, sure.